ksqldb-cli/src/main/java/io/confluent/ksql/cli/Options.java

```diff
@@ -111,6 +113,12 @@
           + "defaults to TABULAR)")
   private String outputFormat = OutputFormat.TABULAR.name();
 
+  @SuppressWarnings("unused") // Accessed via reflection
+  @Option(
+      name = {EXECUTE_OPTION, EXECUTE_SHORT_OPTION},
+      description = "Execute a command and quit.")
```

Comment:
nit: the user may not understand "command" - maybe "run one or more ksql statements and quit."?

Reply:
Done.